docs/draft-release-notes.md

```diff
@@ -1,5 +1,6 @@
 ## Release notes for next release:
 
 ### New feature:
-
+- Client Secret field validation in failover service: We introduce a less restrictive validation. The main validation is on the supplier's side, and we just check if the field is empty or too long.
```

Comment:
Are there any edits to these release notes that might be appropriate now that we have new guidance on writing release notes?